### PR TITLE
fix: generate googleapis from git

### PR DESCRIPTION
**Pull request by ericwenn**
Buf have removed the majority of this package from the main registry...

### CODE REVIEW

File: .sage/proto.go [hunk 1]
```diff
@@ -15,7 +15,7 @@ const (
 	// Version of buf.build/googleapis/googleapis to generate from.
 	// Versions can be found here:
 	// https://buf.build/googleapis/googleapis/history
-	googleapisRef = "f3590c56d388417ebbedefae77ac12bf"
+	googleapisRef = "0487c63ac8292e507e0f3fecb3c01efdadea4967"
```
Comment (thall) [inline .sage/proto.go:18]: update comment, github repo is used now

**Review by thall — APPROVED** [review]

File: .sage/proto.go (hunk 1 above)
Comment (ericwenn) [reply .sage/proto.go]: On a second look it seems like a bad idea to use a specific ref in googleapis repository.
In that repository there are no tags or release branches, so must use a specific commit or master.

When using a specific commit the entire git history will eventually have to be cloned to check out the commit.

--

Will generate from master, and see if it is a problem.

File: .sage/proto.go [hunk 2]
```diff
@@ -67,7 +60,7 @@ func (Proto) BufGenerateGoogleapis(ctx context.Context) error {
 	cmd := sgbuf.Command(
 		ctx,
 		"generate",
-		"buf.build/googleapis/googleapis:"+googleapisRef,
+		"https://github.com/googleapis/googleapis.git#branch=master",
```
Comment (radhus) [inline .sage/proto.go:63]: Couldn't this lead to an unstable CI, as there could be changes (e.g. added fields) to the files that we generate from here 🤔

**Review by radhus — APPROVED** [review]

File: .sage/proto.go (hunk 2 above)
Comment (ericwenn) [reply .sage/proto.go:63]: Yes, it will. See my comment above.

File: .sage/proto.go (hunk 1 above)
Comment (ericwenn) [reply .sage/proto.go:19]: No, will generate specific commit with more fetch depth.

**Pull request merged**